assembl/static2/js/app/components/debate/brightMirror/fictionBodyToolbar.jsx

```diff
@@ -59,7 +96,7 @@ const SentimentMenu = ({ postId, mySentiment, screenWidth, position, sentimentCo
                 return result;
               }, [])}
             </div>
-            <div className="txt" style={{ paddingLeft: `${(count + 2) * 6}px` }}>
+            <div className="txt" style={{ paddingLeft: `${(count + 4) * 6}px` }}>
```

Comment:
Don't you get huge padding if count is really high ? How about using count.toString().length ? Because I guess this is about the length of the number ?

Reply:
If it gets messy we will fix it: shouldn't cause any problem though 👍